### ISSUE TITLE
Update CharacterParser to fit new json format

### ISSUE THREAD

**Issue by wsowens** [ufosc/MuddySwamp#8]
We have now added a "start" field to the CharacterClass json. See ExampleClass.json as an example:
```
{
    "name" : "ExampleClass",
    "frequency" : 1,
    "start" : "Turlington Plaza",
    "path" : "scripts/ExampleClass.py"
}
```
The CharacterParser should attempt to grab this field, and create a Dependency to be resolved based on this field. Resolve function might look like:
```
def resolve(class=imported_class, location_name=json_data["start"]):
    class.start = library[Location][location_name]
```
Refer to the dependency creation process in LocationParser for more details.

**Comment by wsowens** [comment]
Note:
This "start" field will enable us to define different spawning locations for different character types. So, wizards might spawn in the Dark Tower, while alligators might spawn by a pond.

**Comment by hjarrell** [comment]
This should be closed by 672f36e32b9501092f22f495e5b9ebd1a025801c

**Comment by wsowens** [comment]
Issue solved on develop-inventory
